components/base/BaseCollectionForm.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <p class="title is-size-3">
+      {{ $t(label) }}
+    </p>
+    <slot name="header"></slot>
+    <b-field>
+      <Auth />
+    </b-field>
+
+    <MetadataUpload
+      v-model="vFile"
+      label="Drop collection logo here or click to upload or simple paste image from clipboard. We support various media types (PNG, JPEG, GIF, SVG)"
+      expanded
+      preview
+      accept="image/png, image/jpeg, image/gif, image/svg+xml, image/svg" />
+
+    <BasicInput
+      v-model="vName"
+      :label="$t('mint.collection.name.label')"
+      :message="$t('mint.collection.name.message')"
+      :placeholder="$t('mint.collection.name.placeholder')"
+      expanded
+      spellcheck="true" />
+
+    <slot name="main"></slot>
+
+    <BasicInput
+      v-model="vDescription"
+      maxlength="500"
+      type="textarea"
+      spellcheck="true"
+      :class="descriptionClass"
```

Comment:
```suggestion
      :class="{'mt-5': protectiveMargin}"
```
for more consistency